vmware-base/src/com/cloud/hypervisor/vmware/mo/VmdkFileDescriptor.java

```diff
@@ -216,10 +226,12 @@ public String getAdapterType() {
                 }
             }
         } finally {
-            if (in != null)
+            if (in != null) {
```

Comment:
While I know it not in scope of the PR, would it be possible to convert the try block into a try with resources? It would remove the need for this finally clause.